src/inspect_ai/tool/_tools/_bash_session.py

```diff
@@ -52,13 +53,14 @@ def viewer(tool_call: ToolCall) -> ToolCallView:
 
 
 @tool(viewer=code_viewer("bash", "command"))
-def bash_session(timeout: int | None = None) -> Tool:
+def bash_session(*, timeout: int | None = None, instance: str | None = uuid()) -> Tool:
```

Comment:
I have no problem with forcing all of the params to be keyword params. I'm curious about any broad principles I could adopt. For example, what motivated you to add it in this case?

Reply:
Just for clarity at the call site about what is being set. Also so we can arbitrarily re-order the parameters if/when the list grows over time.